lib/ui/shared/styles.dart

```diff
@@ -50,6 +50,25 @@ class AppTextStyles {
     fontWeight: FontWeight.w400,
     fontSize: 12,
   );
+  static TextStyle messageText = GoogleFonts.lato(
```

Comment:
Were this styles added to the design or just for the UI's you built ?

Reply:
I added this for the UI I built, to serve as the text style for all the messages handled by the Thread cards.